clams/restify/__init__.py

```diff
@@ -166,8 +166,8 @@ def cast(self, args):
             if k in self.param_spec:
                 if self.param_spec[k] == bool:
                     casted[k] = self.bool_param(v)
-            else:
-                casted[k] = v
+                else:
```

Comment:
I think we actually need to take care of another problems as well. Concretely, we need to add other conditions to that exact `if` block to cast other types of parameter values. `str`-based parameter types passed from the app metadata is converted to python types via [this line](https://github.com/clamsproject/clams-python/blob/master/clams/app/__init__.py#L28), but in the `Caster` class, the parameter values (passed from url query strings) are not converted to python data types except for `bool`-typed parameters.